README.md

```diff
@@ -52,6 +52,10 @@ Note: GKE or Anthos natively support injecting workload identity for pods.  This
         #           The value must be one of 'gcloud'(default) or 'direct'.
         #           Refer to the next section for 'direct' injection mode
         cloud.google.com/injection-mode: "gcloud"
+
+        # optional: Defaults to value inside `service-account-email`
+        #           
+        cloud.google.com/project: "12345"
```

Comment:
It's not obvious whether this needs to be the project ID or the project Number.

If it's normally parsed out of the SA then it seems it should be the Project ID, but the example here has a number?!

Unless "SA" means Kubernetes SA, and workload identity provider annotation?

Either way, it would be better if the annotation name was explicit.

Reply:
I am not sure whether it matters. I'm not working with  GCP for a while already so I can't test, but I remember most of the tools calling the field `project` and accepting both numerical IDs and project names.